docs/common/istio-ambient-mode.md

```diff
@@ -0,0 +1,354 @@
+    bookinfo     productpage-v1-57ffb6658c-vmb6z 10.129.0.48  ip-10-0-0-241.ec2.internal None     TCP
+    bookinfo     ratings-v1-794744f5fd-wm798     10.129.0.44  ip-10-0-0-241.ec2.internal None     TCP
+    bookinfo     reviews-v1-67896867f4-4h9j6     10.129.0.45  ip-10-0-0-241.ec2.internal None     TCP
+    bookinfo     reviews-v2-86d5db4bd6-dbw4f     10.129.0.46  ip-10-0-0-241.ec2.internal None     TCP
+    bookinfo     reviews-v3-77947c4c78-r54c9     10.129.0.47  ip-10-0-0-241.ec2.internal None     TCP
+...
+```
+
+After adding it to the Ambient mesh, you should see a different result using HBONE protocol.
+
+```bash
+$ istioctl -n ztunnel ztunnel-config workloads
+    NAMESPACE    POD NAME                        ADDRESS      NODE                       WAYPOINT PROTOCOL
+    bookinfo     details-v1-6cd6d9df6b-mddv2     10.129.0.43  ip-10-0-0-241.ec2.internal None     HBONE
+    bookinfo     productpage-v1-57ffb6658c-vmb6z 10.129.0.48  ip-10-0-0-241.ec2.internal None     HBONE
+    bookinfo     ratings-v1-794744f5fd-wm798     10.129.0.44  ip-10-0-0-241.ec2.internal None     HBONE
+    bookinfo     reviews-v1-67896867f4-4h9j6     10.129.0.45  ip-10-0-0-241.ec2.internal None     HBONE
+    bookinfo     reviews-v2-86d5db4bd6-dbw4f     10.129.0.46  ip-10-0-0-241.ec2.internal None     HBONE
+    bookinfo     reviews-v3-77947c4c78-r54c9     10.129.0.47  ip-10-0-0-241.ec2.internal None     HBONE
+...
+```
+
+- [Verify mutual TLS is enabled](https://istio.io/latest/docs/ambient/usage/verify-mtls-enabled/)
+
+You can also validate mTLS from ztunnel logs to confirm mTLS is enabled.
+
+```bash
+$ kubectl -n ztunnel logs -l app=ztunnel | grep -E "inbound|outbound"
+
+2025-01-23T05:07:25.806642Z	info	access	connection complete	src.addr=10.129.0.48:40978 src.workload="productpage-v1-57ffb6658c-vmb6z" src.namespace="bookinfo" src.identity="spiffe://cluster.local/ns/bookinfo/sa/bookinfo-productpage" dst.addr=10.129.0.43:15008 dst.hbone_addr=10.129.0.43:9080 dst.service="details.bookinfo.svc.cluster.local" dst.workload="details-v1-6cd6d9df6b-mddv2" dst.namespace="bookinfo" dst.identity="spiffe://cluster.local/ns/bookinfo/sa/bookinfo-details" direction="outbound" bytes_sent=283 bytes_recv=358 duration="2ms"
+```
+
+Validate the `src.identity` and `dst.identity` values are correct. They are the identities used for the mTLS communication among the source and destination workloads.
+
+### Cleanup
+
+If you no longer need associated resources, you can delete them by following the steps below.
+
+#### Remove waypoint proxies
+
+```bash
+kubectl label namespace bookinfo istio.io/use-waypoint-
+kubectl delete -n bookinfo Gateway waypoint
+```
+
+#### Remove the namespace from the ambient data plane
+
+```bash
+kubectl label namespace bookinfo istio.io/dataplane-mode-
+```
+
+> **NOTE**:
+> You must remove workloads from the ambient data plane before uninstalling Istio.
```

Comment:
Did you notice any issues when Istio is uninstalled without removing the workloads from ambient?

Reply:
I haven't tried that scenario. This note is mentioned from upstream doc so I copied it from upstream doc:

https://istio.io/latest/docs/ambient/getting-started/cleanup/#remove-the-namespace-from-the-ambient-data-plane